extensions/sql-migration/src/dialog/skuRecommendationResults/getAzureRecommendationDialog.ts

```diff
@@ -332,16 +331,11 @@ export class GetAzureRecommendationDialog {
 		this.migrationStateModel._skuRecommendationPerformanceDataSource = this._performanceDataSource;
 		switch (this.migrationStateModel._skuRecommendationPerformanceDataSource) {
 			case PerformanceDataSourceOptions.CollectData: {
-				// TO-DO: expose the rest of these in the UI
-				const perfQueryIntervalInSec = 3;
-				const staticQueryIntervalInSec = 30;
-				const numberOfIterations = 5;
-
 				await this.migrationStateModel.startPerfDataCollection(
 					this.migrationStateModel._skuRecommendationPerformanceLocation,
```

Comment:
do we need to pass in the this.migrationStateModel._skuRecommendationPerformanceLocation value?

Reply:
Good point, will clean it up in the next PR